Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Post Item/Feed Post.swift

```diff
@@ -35,115 +35,44 @@ struct FeedPost: View
     @State private var isShowingEnlargedImage: Bool = false
     
     // swipe-to-vote
-    @State var dragPosition: CGSize = .zero
-    @State var prevDragPosition: CGFloat = .zero
-    @State var dragBackground: Color = .systemBackground
-    @State var leftSwipeSymbol: String = "arrow.up"
-    @State var rightSwipeSymbol: String = "arrowshape.turn.up.left"
+    @Binding var isDragging: Bool
     
     // in-feed reply
     @State var replyIsPresented: Bool = false
     @State var replyContents: String = ""
     @State var replyIsSending: Bool = false
-   
     
     var body: some View {
-            ZStack {
-                dragBackground
-                HStack(spacing: 0) {
-                    Image(systemName: leftSwipeSymbol)
-                        .font(.title)
-                        .frame(width: 20, height: 20)
-                        .foregroundColor(.white)
-                        .padding(.horizontal, 20)
-                    Spacer()
-                    Image(systemName: rightSwipeSymbol)
-                        .font(.title)
-                        .frame(width: 20, height: 20)
-                        .foregroundColor(.white)
-                        .padding(.horizontal, 20)
+        VStack(spacing: 0) {
+            postItem
+                .background(Color.systemBackground)
+                .contextMenu {
+                    Button("hit me!") {
+                        print("hit")
+                    }
+                }
+                .addSwipeyActions(isDragging: $isDragging,
```

Comment:
Recommend using a config object.
`struct SwipeyActionConfig` and that way you can store it as a preference easier without it being in the swiftUI code directly.

Reply:
That's a good idea. I'll scope it for a follow-up PR--want to get this one in quickly